lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,28 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+// TO-DO: once existing lms $blue is removed, change $cms-blue to $blue. 
+$cms-blue: rgb(0, 159, 230);
+$blue-l1: tint($cms-blue,20%);
```

Comment:
The idea for using a temporary "cms-blue" is to allow us to add the other blue shades without having to proof the existing $blue in the LMS. 

The only blue sass variables that were overridden with this change are $blue-d1, $blue-d2, and $blue-d4, which are used in styling for xmodule tabs (I believe some fallout from ORA2 not being able to fully inherit / use existing buttons / controls. These blues should actually have been based off of cms-blue, so the slight change in shades of blue here relative to the old lms $blue is a small improvement. 

The sass in the static/certificates/ directory still uses the old blue, so I didn't migrate that over but we can do that cleanup once we turn back to that view in the coming sprints.